mlflow/system_metrics/metrics/disk_monitor.py

```diff
@@ -0,0 +1,21 @@
+"""Class for monitoring disk stats."""
+
+import os
+
+import psutil
+
+from mlflow.system_metrics.metrics.base_metrics_monitor import BaseMetricsMonitor
+
+
+class DiskMonitor(BaseMetricsMonitor):
+    """Class for monitoring disk stats."""
+
+    def __init__(self, name="disk"):
+        super().__init__(name)
+
+    def collect_metrics(self):
+        # Get disk usage metrics.
+        disk_usage = psutil.disk_usage(os.sep)._asdict()
+        self._metrics["disk_usage_percentage"].append(disk_usage["percent"])
+        self._metrics["disk_usage_megabytes"].append(int(disk_usage["used"] / 1e6))
+        self._metrics["disk_available_megabytes"].append(int(disk_usage["free"] / 1e6))
```

Comment:
same comment on this part
https://github.com/mlflow/mlflow/pull/9557/files#r1330882399